GSA-TTS/federal-website-standards
# Update standard page layouts

## Context
Current page template is missing H1. We also need a "list" template to show/filter/sort all standards.

## What changed
* Printed pages titles as an H1 on the page.html layout template.
* Created a collection.html template. It lists all files of tag type `standard` in the USWDS [collection format](https://designsystem.digital.gov/components/collection/).
* Created a cards.html template. It lists all files of tag type `standard` in the USWDS [card format](https://designsystem.digital.gov/components/card/).
* Separated standards into individual markdown files. Marked hidden standards as `eleventyExcludeFromCollections` in the front matter. (Think of these as "drafts").
* Created a summary-box component. We can configure this to show key information on longer pages. See the USWDS [guidance on this component](https://designsystem.digital.gov/components/summary-box/).


## How to verify the change
1. Click on any page. Make sure the H1 matches the `title` front matter in its corresponding markdown file.
2. Click on Federal web standards link in the top navigation. Should list teaser view of all published (not draft) standards.
3. Click on the teaser title to see a full standard page. ➕  **Bonus!** [The HTML Page Title](https://federalist-ce0b03c9-1a0d-440d-8d5d-6278c649a788.sites.pages.cloud.gov/preview/gsa-tts/federal-web-standards/cannandev/update-templates/standards/html-page-title/) page shows a summary box.

## Additional info
[🔗 Preview link](https://federalist-ce0b03c9-1a0d-440d-8d5d-6278c649a788.sites.pages.cloud.gov/preview/gsa-tts/federal-web-standards/cannandev/update-templates/)
